src/lint/rules/tests/indentation/mod.rs

```diff
@@ -0,0 +1,34 @@
+mod in2;
+mod in3;
+mod in4;
+mod in5;
+mod in9;
+mod in10;
+
+use crate::lint::rules::tests::common::assert_snippet;
+use crate::lint::LintCfg;
+use crate::lint::LongLineOptions;
+use crate::lint::rules::instantiate_rules;
+use std::convert::TryInto;
+
+//  Line length can be configured to a maximum
+//(defaults to 80, feature disabled)
+static LINE_LENGTH_INCORRECT: &str = "
+param some_parameter_name_in_this_device = some_long_name_bank.some_long_name_group.SOME_REGISTER_NAME;
+";
+#[test]
+fn line_length_incorrect() {
+    let mut cfg = LintCfg::default();
+    let mut rules = instantiate_rules(&cfg);
+    assert_snippet(LINE_LENGTH_INCORRECT, 1, &rules);
+    // Test rule disable
+    cfg.long_lines = None;
+    rules = instantiate_rules(&cfg);
+    assert_snippet(LINE_LENGTH_INCORRECT, 0, &rules);
+    // Test lower max_length
```

Comment:
Should probably have the test be three lines. one short, lone medium length, and one really long. then test how many lines you hit with each setting (so 0 on disable or overly long setting, 1 with regular, 2 with a slightly shorter setting)

Reply:
Will improve this test when implementing line breaks in https://github.com/intel/dml-language-server/issues/73